src/formatDate.js

```diff
@@ -51,6 +51,38 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const fromFormatObj = {};
+  const fromF = fromFormat[3];
+  const toF = toFormat[3];
+  const dateArr = date.split(fromF);
+  const res = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    fromFormatObj[fromFormat[i][0]] = dateArr[i];
+  }
+
+  if (fromFormat.find(el => el.startsWith('Y')).length === 2) {
```

Comment:
save the result of method `find` in a variable, in that way your code gonna look cleaner